routes/api/meme.js

```diff
@@ -3,38 +3,34 @@ var db = require('../../models');
 module.exports.addMeme = function(req, res) {
     //TODO create meme
     // res.sendStatus(503)
-    db.meme_texts.create({text:req.body.top}).then(function(top_text) {
-        db.meme_texts.create({text:req.body.bottom}).then(function(bottom_text) {
-                db.memes.create().then(function(meme) {
-
-                    // the image here should be the id of an already existing/ newly uploaded image,
-                    // i.e. this image id should already be saved in db
-                    meme.setImage(req.body.image);
-                    meme.setTopText(top_text);
-                    meme.setBottomText(bottom_text);
-                    res.json({meme: meme});
-                }).catch(function(err) {
-                    console.log("meme fail");
-                    res.status(400).send(err);
-                });
-        }).catch(function(err) {
-            console.log("bottom text fail");
-            res.status(400).send(err);
-        });
+
+    
+    db.memes.create({toptext:req.body.top,bottomtext:req.body.bottom}).then(function(meme) {
+
+        // the image here should be the id of an already existing/ newly uploaded image,
+        // i.e. this image id should already be saved in db
+        meme.setimage(req.body.image);
```

Comment:
Not sure if this works. The correct method might be `setImage()` instead of `setimage()`